lib/transaction.js

```diff
@@ -85,7 +85,14 @@ Transaction.begin = function(connector, options, cb) {
       return cb(err);
     }
     var tx = connection;
-    if (!(connection instanceof Transaction)) {
+
+    // When the connector and juggler node module have different version of this module as dependencies,
```

Comment:
typo: `node module` -> `node modules`, `version` -> `versions`, and `as dependencies` -> `as a dependency`